src/common.h

```diff
@@ -1,2 +1,6 @@
 
 #pragma once
+
+#define ERR "ERR"
```

Comment:
`ERR` is a little broad-spectrum when defined in a header. Maybe `TS_ERR`?

Reply:
fixed